themes/default/content/challenge/startup-in-a-box.md

```diff
@@ -41,6 +41,10 @@ meta_desc: |
 
 ### Challenge
 
+Follow along with the steps below or watch the video to complete the challenge.
```

Comment:
Generally, we don't use directional language in case someone is using a screen reader.

```suggestion
Follow along with the steps outlined on this page, or watch this video to complete the challenge.
```